packages/gasket-fetch/test/browser.spec.js

```diff
@@ -1,19 +1,25 @@
-import { describe, it } from 'mocha';
-import assume from 'assume';
-
+/* eslint-disable jest/expect-expect, jest/no-done-callback */
+import 'whatwg-fetch';
 import fetch from '../lib/browser';
 const { AbortController, Request, Headers, Response } = fetch;
 import { assertExports, assertGet, assertAbort, assertPost } from './utils';
 
 describe('fetch is available in browsers', function () {
+
   it('exposes default classes', assertExports(Request, Headers, Response));
-  it('can fetch resources', assertGet(fetch));
-  it('fetch can POST to API', assertPost(fetch));
+  it('can fetch resources', (done) => {
+    assertGet(fetch);
+    done();
+  });
+  it('fetch can POST to API', (done) => {
+    assertPost(fetch);
+    done();
+  });
```

Comment:
Go ahead and update these to be async, so you can remove the disable `jest/no-done-callback` above.
```suggestion
  it('can fetch resources', async () => {
    await assertGet(fetch);
  });
  it('fetch can POST to API', async () => {
    await assertPost(fetch);
  });
```

Reply:
This doesn't work that's why it's "done" this way.
![Screen Shot 2023-01-18 at 9 41 29 AM](https://user-images.githubusercontent.com/105235096/213255012-5a616115-d85e-4403-9682-6adc030eda06.png)

Reply:
You still have done as an argument to the callback, you need to remove that.